### PR TITLE
Fix #10238 Change Math.random with an alternative solution 

### LINKED ISSUE #10238: Change Math.random usages with an alternative
## Description

It was requested to substitute every usage of Math.random with an alternative like window.crypto.getRandomValues
for that we need to update two branches the 
- c039_math_random 
- master

**What kind of improvement you want to add?** (check one with "x", remove the others)
- [ ] Minor changes to existing features
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

### PR DESCRIPTION
**Pull request by MV88**
## Description

Fix #10238

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [x] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

 Fix #10238

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/utils/cesium/MathUtils.js [hunk 1]
```diff
@@ -180,3 +180,21 @@ export function computeGeodesicCoordinates(positions, height) {
...
+    const int = window.crypto.getRandomValues(new Uint32Array(1))[0];
+    return int / 2 ** 32;
+};
+
+/**
+ * Return integers in the range of [min, max)
+ *
+ * @todo check that min is <= max.
+ */
+export const randomInt = function(max = 123456789, min = 0) {
```
Comment (offtherailz) [inline web/client/utils/cesium/MathUtils.js:197]: 123456789 is not the max number. Use Number.MAX_SAFE_INTEGER

**Comment by MV88** [comment]
@offtherailz please review it again since i have fixed the paths of the functions.

for eslint I tried but I was not able to integrate the singular rule, or to make it to be treated as error.
probably i was not correctly setup the project, what I tried was:
- npm install --save-dev eslint-plugin-security-node on root
- edit node_modules\@mapstore\eslint-config-mapstore\index.js by adding the plugins (and the single rule  with **"detect-insecure-randomness",2** not working))
- edit package.json by adding the extends required
- adding some code with math.random() 
- reloading window

result: 
i was getting this
image

and by removing  the rule from index.js of node_modules eslint i got only the highlight as suggestion/warning

in [this](https://github.com/MV88/MapStore2/tree/detect-insecure-randomness) branch there is the partial setup, then you have to edit locally the eslint files used by mapstore

**Comment by offtherailz** [comment]
I renamed the new file as `RandomUtils` to avoid confusion (in fact it contains only random number generation functionalities).

File: web/client/utils/__tests__/RandomUtils-test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+import expect from 'expect';
+import {
+    randomFloat,
+    randomInt
+} from '../RandomUtils';
+
+describe('Test MathUtils', () => {
```
Comment (offtherailz) [inline web/client/utils/__tests__/RandomUtils-test.js:14]:
```suggestion
describe('Test RandomUtils', () => {
```

**Comment by offtherailz** [comment]
Let's skip the eslint check for the moment.

**Pull request merged**